tests/backend/mxnet/__init__.py

```diff
@@ -25,6 +25,9 @@ def allclose(a, b, rtol=1e-4, atol=1e-4):
 def randn(shape):
     return nd.random.randn(*shape)
 
+def rand(shape):
+    return nd.random.randint(0, 1, *shape)
```

Comment:
Why randint?

Reply:
> Why randint?

Oops. Just clean it out.
All test will use randn